docker_entrypoint.sh

```diff
@@ -1,3 +1,27 @@
 #!/bin/sh
 
-exec tini hello-world
+set -e
+
+ITCHYSATS_USER=itchysats
+ITCHYSATS_PASSWORD=$(yq e '.password' /root/start9/config.yaml)
```

Comment:
Note: `/root/start9/config.yaml` is the dir within the container. The mounted directory is `/embassy-data/package-data/volumes/itchysats/data/main/start9/config.yaml`

Yes, the password is stored in plaintext in that file, but in order to read that file you would need access to the `embassy`.

I agree that keeping the password within ItchySats will be better; this can be configured once we have the next release out. We can then remove the configuration once the next release is out!